arangod/Aql/DocumentProducingNode.cpp

```diff
@@ -39,14 +40,14 @@
 using namespace arangodb::aql;
 
 namespace {
-std::string_view const filterKey("filter");
-std::string_view const maxProjectionsKey("maxProjections");
-std::string_view const producesResultKey("producesResult");
+constexpr std::string_view filterKey("filter");
```

Comment:
There is no need to make this variable static, because it is in the anonymous namespace of the .cpp file already.
I can move it to StaticStrings though.